src/components/sbb-tag-group/sbb-tag-group.tsx

```diff
@@ -13,43 +22,116 @@ import { isValidAttribute } from '../../global/helpers/is-valid-attribute';
 export class SbbTagGroup implements ComponentInterface {
   @Element() private _element: HTMLElement;
 
-  public tags: HTMLSbbTagElement[];
+  /**
+   * If set multiple to false, the selection is exclusive and the value is a string (or null).
+   * If set multiple to true, the selection can have multiple values and therefore value is an array.
+   *
+   * Changing multiple during run time is not supported.
+   */
+  @Prop() public multiple = false;
 
-  private _getTags(): HTMLSbbTagElement[] {
-    return Array.from(this._element.querySelectorAll('sbb-tag')) as HTMLSbbTagElement[];
+  /**
+   * Value of the sbb-tag-group.
+   * If set multiple to false, the value is a string (or null).
+   * If set multiple to true, the value is an array.
+   */
+  @Prop({ mutable: true }) public value: string | string[] | null = null;
+
+  @Watch('value')
+  public valueChanged(value: string | string[] | null): void {
+    if (Array.isArray(value) && !this.multiple) {
+      console.warn(
+        'Trying to set array value for sbb-tag-group but multiple mode is not activated.',
+        value
+      );
+      return;
+    } else if (!Array.isArray(value) && this.multiple) {
+      try {
+        // If possible, try to parse JSON and restart to assign value.
+        this.value = JSON.parse(value);
+      } catch {
+        console.warn(
+          'Expected value to be an array because sbb-tag-group multiple mode is activated.',
+          value
+        );
+      }
+      return;
+    }
+
+    const tags = this._tags();
+
+    if (this.multiple) {
+      tags.forEach((tag) => (tag.checked = value.includes(tag.value)));
+    } else {
+      tags.forEach((tag) => (tag.checked = tag.value === value));
+    }
   }
 
-  private _onTagSlotChange(): void {
-    this.tags = this._getTags();
+  private _ensureOnlyOneTagSelected(): void {
+    if (this.multiple) {
+      return;
+    }
+
+    // Ensure only one tag checked
+    const checkedTags = this._tags().filter((tag) => tag.checked);
+    if (checkedTags.length > 1) {
```

Comment:
Nitpick; The if check is not strictly necessary, as you could just call slice(1)